app/src/main/java/com/example/tasky/login/presentation/LoginScreen.kt

```diff
@@ -75,59 +81,74 @@ fun LoginScreen() {
                     horizontalAlignment = Alignment.CenterHorizontally,
                 ) {
                     Spacer(modifier = Modifier.size(50.dp))
-                    TaskyTextField()
+
+                    TaskyTextField(
+                        value = loginState.emailValue,
+                        imeAction = Companion.Next,
+                        isValidEmail = loginState.isValidEmail
+                    ) { emailValue ->
+                        onLoginEvent(LoginEvent.OnEmailValueChanged(emailValue))
+                    }
                     Spacer(modifier = Modifier.size(16.dp))
-                    TaskyTextField()
+
+                    PasswordTaskyTextField(
```

Comment:
I just noticed

This component could be named "TaskyPasswordTextField", to maintain the same name structure